services/storage/src/simcore_service_storage/modules/celery/worker_main.py

```diff
@@ -0,0 +1,88 @@
+"""Main application to be deployed in for example uvicorn."""
+
+import asyncio
+import logging
+import threading
+
+from asgi_lifespan import LifespanManager
+from celery import Celery
+from celery.signals import worker_init, worker_shutdown
+from servicelib.background_task import cancel_wait_task
+from servicelib.logging_utils import config_all_loggers
+from simcore_service_storage.modules.celery.utils import set_celery_worker
+
+from ...core.application import create_app
+from ...core.settings import ApplicationSettings
+from .common import create_app as create_celery_app
+from .example_tasks import sync_archive
+from .worker import CeleryTaskQueueWorker
+
+_settings = ApplicationSettings.create_from_envs()
+
+# SEE https://github.com/ITISFoundation/osparc-simcore/issues/3148
+logging.basicConfig(level=_settings.log_level)  # NOSONAR
+logging.root.setLevel(_settings.log_level)
+config_all_loggers(
+    log_format_local_dev_enabled=_settings.STORAGE_LOG_FORMAT_LOCAL_DEV_ENABLED,
+    logger_filter_mapping=_settings.STORAGE_LOG_FILTER_MAPPING,
+    tracing_settings=_settings.STORAGE_TRACING,
+)
+
+_logger = logging.getLogger(__name__)
+
+
+@worker_init.connect
+def on_worker_init(sender, **_kwargs):
+    def _init_fastapi():
+        loop = asyncio.new_event_loop()
```

Comment:
you create a new look, who is it closed/cleaned up?

Reply:
on shutdown